references/cli/addon.go

```diff
@@ -396,12 +396,13 @@ func statusAddon(name string, ioStreams cmdutil.IOStreams, cmd *cobra.Command, c
 	if err != nil {
 		return err
 	}
-	status, err := pkgaddon.GetAddonStatus(context.Background(), k8sClient, name)
+
+	status, str, err := generateAddonInfo(k8sClient, name)
 	if err != nil {
 		return err
 	}
 
-	fmt.Print(generateAddonInfo(name, status))
+	fmt.Print(str)
```

Comment:
shall we remove this line? It's more like a debug message here.

Reply:
It will print the actual outputs of `vela addon status`, which is returned as a string in `str`. I will refactor the code to make it clearer.